requirements.txt

```diff
@@ -1 +1 @@
-ansible
\ No newline at end of file
+ansible==2.2.1.0
```

Comment:
Are you planning to maintain this for every minor bump? Ansible 2.2.2 was recently released.